### PR TITLE
Feature ball tracking

### PR DESCRIPTION
**Pull request by maaarsas**
Pridėjau kamuoliuko trackinimą. Šiuo metu ant kiekvieno frame'o tiesiog nupiešia kontūrus ant kamuoliuko, kur jį randa. 
Spalva kol kas į hardcodinta.

### CODE REVIEW

**Comment by DanielRitvas** [comment]
What if we create a branch called develop and merge all implemented features there before putting it into master branch? Lets keep only fully developed functionality in master?

**Comment by maaarsas** [comment]
Well, the project is quite small so I doubt this is neccessary. But we can do it like that, I'm totally fine with it.

**Review by DanielRitvas — APPROVED** [review]

File: foosballv2s/MovementDetector.cs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        // 3) All unneeded colors are filtered out, only pixels with given color are left
+        // 4) After this step the picture is in grayscale where the ball appears white and all other
+        //    pixels are black
+        // 5) Circles are detected in the grayscale image
+        // 6) The proccessed frame with circles drawn on it is shown in a window
+        public bool DetectBall()
+        {
+            // Currently the color is hardcoded here. Needs to be detected somewhere else
+            // and passed here as parameters
+            int t1min = 170, t1max = 4, t2min = 120, t2max = 255, t3min = 120, t3max = 255; // color in HSV
```
Comment (IndirasM) [inline foosballv2s/MovementDetector.cs:30]: As I understand it, this part will be left out in the main project as we will opt for picture color recognition. But the question remains how large will we leave the opening for color changes.

**Review by IndirasM — APPROVED** [review]
Good job, well commented and understandable code.

**Comment by Radvila** [comment]
Nicely commented. I suggest to create unit test on tracking high speed ball movement

**Comment by Radvila** [comment]
Create unit test on tracking high speed ball movement

**Review by Radvila — APPROVED** [review]

File: foosballv2s/MovementDetector.cs (hunk 1 above)
Comment (maaarsas) [reply foosballv2s/MovementDetector.cs:30]: Probably we'll see what range of colors should be picked for best results as we continue to develop further.

**Review by ievara — APPROVED** [review]

**Pull request merged**